src/router.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+const express = require('express');
+const db = require('./db');
+
+const router = express.Router();
+
+
+// register
+router.post('/register', (req, res, next) => {
+  res.sendStatus(200);
+  next();
+});
+
+// login
+
+router.post('/login', (req, res, next) => {
+  res.sendStatus(200);
+  next();
+});
+
+// Gets all items
+router.get('/allproducts', (req, res, next) => {
+  res.sendStatus(200);
+  next();
+});
+
+
+// Gets one item by ID
+router.get('/product/:id', (req, res, next) => {
+  res.sendStatus(200);
+  next();
+});
+
+// Creates an item
+router.post('/create', (req, res, next) => {
```

Comment:
change the route to /product. Create is implied by the http method "post"

Reply:
Hey Thomas thank you for your feedback i was actually planning on changing it. It was left unchanged in error.